lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+const DEFAULT_MANIFEST_PREFS = {
+  "social.manifest.twitter-com": {
+    "author": "Twitter, Inc.",
+    "description": "Use your Firefox Twitter Share to quickly author tweets about the pages you discover across the web.  We shorten URLs for quick sharing using the t.co link shortener.",
+    "homepageURL": "https://twitter.com",
+    "icon32ULR": "images/twitter/icon32.png",
```

Comment:
Also, should we base64 inline the icon32.png like we do for `icon64URL` and `iconURL` below?